biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -99,6 +101,15 @@ public Version getCapabilityVersion() {
 		return capabilityVersion;
 	}
 
+	public static EE highestFromVersion(String versionString) {
```

Comment:
I would suggest this API return `Optional<EE>`. This lets the caller decide how to deal with optionality vs. returning null.